src/wallet/rpc/transactions.cpp

```diff
@@ -417,6 +421,10 @@ static std::vector<RPCResult> TransactionDescriptionString()
            }},
            {RPCResult::Type::STR_HEX, "replaced_by_txid", /*optional=*/true, "Only if 'category' is 'send'. The txid if this tx was replaced."},
            {RPCResult::Type::STR_HEX, "replaces_txid", /*optional=*/true, "Only if 'category' is 'send'. The txid if this tx replaces another."},
+           {RPCResult::Type::ARR, "mempoolconflicts", "Transactions that directly conflict with either this transaction or an ancestor transaction",
```

Comment:
Not for this PR just so you don't have to re-touch it but it would be good to describe the difference between `walletconflicts` and `mempoolconflicts` inside the help text. `walletconflicts` description is quite vague.